.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,23 +5,23 @@
 <!-- Make sure that your PR is not a duplicate -->
 
 #### Pre-Submission Checklist
-<!-- Go over all points below, and put an `x` in all the boxes that apply. -->
-<!-- All points should be checked, otherwise, read the CONTRIBUTING guidelines from above-->
+<!-- Go over all points below, and after creating the PR click on all the check boxes that apply. -->
+<!-- All points should be verified, otherwise, read the CONTRIBUTING guidelines from above-->
 <!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
 - [ ] Your pull request targets the `staging` branch of FreeCodeCamp.
 - [ ] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
 - [ ] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) them into one commit).
 - [ ] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
 
 #### Type of Change
-<!-- What type of change does your code introduce? Put an `x` in the box that applies. -->
+<!-- What type of change does your code introduce? After creating the PR click on all the check boxes that apply. -->
 - [ ] Small bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds new functionality)
 - [ ] Breaking change (fix or feature that would change existing functionality)
 - [ ] Add new translation (feature adding new translations)
 
 #### Checklist:
-<!-- Go over all points below, and put an `x` in all the boxes that apply. -->
+<!-- Go over all points below, and after creating the PR click on all the check boxes that apply. -->
```

Comment:
~~after creating the PR click on all the check boxes that apply.~~ after creating the PR, **tick** the **checkboxes** that apply.